aws/table_aws_ec2_managed_prefix_list.go

```diff
@@ -22,7 +22,9 @@ func tableAwsEc2ManagedPrefixList(_ context.Context) *plugin.Table {
 		Description: "AWS EC2 Managed Prefix List",
 		List: &plugin.ListConfig{
 			IgnoreConfig: &plugin.IgnoreConfig{
-				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidAction", "InvalidRequest"}),
+				// We are dealing with the UnsupportedOperation error code as a result of the resource's unsupported area.
```

Comment:
@ParthaI This comment is a good start point, but doesn't explain why we need it specifically in this table.

Can you please update the comment so that it explains that we are ignoring the `UnsupportedOperation` error because while the EC2 service supports the `me-south-1` region (at least that's what I think it was, the issue https://github.com/turbot/steampipe-plugin-aws/issues/1576 doesn't have any details at the time of this review comment), listing managed prefix lists is not supported in only that region, so we need to handle that error in this table